assets/js/cell/live_editor/theme.js

```diff
@@ -70,4 +69,6 @@ const theme = {
   },
 };
 
-export default theme;
+const highContrast = {...theme, colors : {...theme.colors, "editor.background": "#060708"}}
```

Comment:
This totally works, but I'd be curious to see if `{ base: "custom", inherit: true }` works :)

Reply:
I got `Error: Illegal theme base!`

Reply:
Ah, now it would be `{ base: "default" }` as we changed the name :)

Reply:
I tried that too, same error :(

Reply:
I see, maybe it requires one of the built in ones then, thanks for checking!